### ISSUE TITLE
학습. 영어 Week2

### ISSUE THREAD

**Issue by lionkang** [muraYlion/FelizLaVida#308]
## 학습내용
- 일정 : 9월 16일(수) 11시
- 패턴3,2 진행
  - 패턴집중훈련, 한국어로 묻고 영어도 답하기

## 숙제
- pattern1, 2 예문작성
  - 패턴별 3개 이상 작성
  - week2 에서 내용 공유
- 주어지는 주제로 영작
  - 전날에 작성한거 카톡으로 공유

### Q1
- Which company are you working for? What kind of business does your company do? Tell me about your company in detail.

#### Intro
- I’m working for “Sports Incorporated,” an agency of sports marketing and advertising, located in Seoul.

#### Main
- We specialize in creating and marketing sport-related items for athletes. Our main clients are Korean sports figures, and our major job is to promote them. We distribute flyers on the streets and create posters of the sports stars. Sometimes, we produce promotional souvenirs and sell them in a shop.

#### Closing
- Basically, I really enjoy working in my company. And my next goal is to work in an international firm.

**Comment by lionkang** [comment]
## Answer
### Intro
- I’m working for “Naver Financial”, esay payament 'NaverPay' and offers a variety of financial services, located in SeoungNam

### Main
- 

### Closing
-

**Comment by lionkang** [comment]
## 패턴연습
### I'm from
- I'm from Jeoungub
- I'm from Gumho-dong
### I'm ready for (to)
- I'm ready to cleaing the room
- I'm ready to eat this weekend in buffet

**Comment by lionkang** [comment]
## 숙제
- 패턴연습 문장 만들어 오기
- 패턴연습 문장 녹음하기
- 작문

**Comment by lionkang** [comment]
### Sept 16
- I stay at home to give relax time
  - I stay at home to relax
- some time to walk
- there were stories about house owners who built their own houses 
- after retire, i go to 강원도 and I want to live 단독주택
  - after my retirement, I want to go to 강원도 and live in a single house
- it helps me release my stress
- I want you to help me
